dts/bindings/pinctrl/nxp,lpc-iocon-pinctrl.yaml

```diff
@@ -125,3 +134,34 @@ child-binding:
         description: |
           Enable I2C mode for a pin. If not present, pin is in GPIO mode. Only
           valid for I2C mode pins
+      nxp,digital-filter:
+        type: int
+        default: 0
+        enum:
+          - 0
+          - 1
+          - 2
+          - 3
+        description: |
+          Enable digital filter. Set number of clock cycles to use as rejection
+          threshold for input pulses. 0 disables the filter. Only valid for
+          lpc11u6x SOC
```

Comment:
note: it looks like you want to have multiple compatibles (with a file containing common fields), this way only lpc11u6x SoCs will get these properties exposed.

Reply:
You're right, this is better. I split the LCP11u6x specific fields into a new file.